### PR TITLE
feat: own dockerfile

### PR DESCRIPTION
**Pull request by olegstepura**
Build own images for different platforms
Generalize Dockerfile, build cross-platform image

### CODE REVIEW

File: .github/workflows/default.yml [hunk 1]
```diff
@@ -8,14 +8,14 @@ jobs:
     env:
       TAG: html-to-pdf
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v2
```
Comment (olegstepura) [inline .github/workflows/default.yml:12]: There were warnings with old actions running on old node version

File: .github/workflows/default.yml [hunk 2]
```diff
@@ -8,14 +8,14 @@ jobs:
...
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
           node-version: '16'
           cache: 'yarn'
       - run: yarn install
       - run: yarn lint --max-warnings 0
       - name: Build container
-        run: docker build --file deploy/test/Dockerfile --tag ${TAG} .
```
Comment (olegstepura) [inline .github/workflows/default.yml:19]: now uses multistage single Dockerfile

File: .yarnrc.yml [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
+  html-to-pdf@*:
+    dependencies:
+      cookie: "*"
+      tslib: "*"
   ws@*:
     dependencies:
       bufferutil: "*"
       utf-8-validate: "*"
 
-yarnPath: .yarn/releases/yarn-3.0.0-rc.2.cjs
```
Comment (olegstepura) [inline .yarnrc.yml:16]: updated yarn

**Review by kilerbee — APPROVED** [review]
LGTM

**Pull request merged**